src/app/shared/services/user/signup.service.ts

```diff
@@ -9,6 +9,8 @@ import { UserService } from './user.service';
 })
 export class SignupService {
     private endpoint = '/api/user/signup';
+    private endpointForgotPassword = '/api/user/forgot-password/token';
+    private endpointResetPassword = '/api/user/forgot-password';
```

Comment:
If you don't use these variables in multiple places, you don't need to factor it out to separate variables, you can use it inline in the code (lines 41 and 47).